src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -179,13 +179,11 @@ private void ReanalyzeOnOptionChange(object sender, OptionChangedEventArgs e)
             {
                 // get off from option changed event handler since it runs on UI thread
                 // getting analyzer can be slow for the very first time since it is lazily initialized
-                var asyncToken = _listener.BeginAsyncOperation("ReanalyzeOnOptionChange");
-
-                // Force analyze all analyzers if background analysis scope has changed.
-                var forceAnalyze = e.Option == SolutionCrawlerOptions.BackgroundAnalysisScopeOption;
```

Comment:
moving this inside might keep things around longer than intended (i.e. if OptionChnagedEventArgs hold onto anything big (or does so in the future)).  Consider keeping this outside.

Reply:
N/M.  I cna see 'e' is used inside as well.

Reply:
Yup.